client/flaschen.py

```diff
@@ -0,0 +1,68 @@
+# -*- mode: python; c-basic-offset: 2; indent-tabs-mode: nil; -*-
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation version 2.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://gnu.org/licenses/gpl-2.0.txt>
+
+import socket
+
+class Flaschen(object):
+  '''A Framebuffer display interface that sends a frame via UDP.'''
+
+  def __init__(self, host, port, width, height, layer=0, transparent=False):
+    '''
+
+    Args:
+      host: The flaschen taschen server hostname or ip address.
+      port: The flaschen taschen server port number.
+      width: The width of the flaschen taschen display in pixels.
+      height: The height of the flaschen taschen display in pixels.
+      layer: The layer of the flaschen taschen display to write to.
+      transparent: If true, black(0, 0, 0) will be transparent and show the layer below.
+    '''
+    self.width = width
+    self.height = height
+    self.layer = layer
+    self.transparent = transparent
+    self.sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    self.sock.connect((host, port))
+    header = ''.join(["P6\n",
+                      "%d %d\n" % (self.width, self.height),
+                      "255\n"])
+    footer = ''.join(["0\n",
+                      "0\n",
+                      "%d\n" % self.layer])
+    self.pixels = bytearray(width * height * 3 + len(header) + len(footer))
```

Comment:
Maybe 'pixels' should now be named 'data' or 'ppm_image' as it is not only containing the pixels but the full ppm.